Kernel/GlobalProcessExposed.cpp

```diff
@@ -622,6 +624,7 @@ class ProcFSOverallProcesses final : public ProcFSGlobalInformation {
         return {};
     }
 };
+#if ARCH(I386) || ARCH(X86_64)
```

Comment:
How are we going to remember to add this when the time comes?

Reply:
I guess when someone tries to run `SystemMonitor`, it'll try to access `/proc/cpuinfo`, which will then fail, and someone can implement this :)

Reply:
Actually, I think it is better to print something to the console, and then return an error. I've implemented that :^)